### ISSUE TITLE
Parser unable to rename nested classes when namespaces are used

### ISSUE THREAD

**Issue by davidsj2** [bytedeco/javacpp#352]
Consider the following class:

```cpp
#ifndef NESTEDTEST_H_
#define NESTEDTEST_H_

namespace com {
namespace company {
class NestedTest
{
public :

  struct Listener {
    virtual ~Listener() {}
  };
};
}}
#endif
```

If a rule is added to the parser as follows:
```java
infoMap.put(new Info("com::company::NestedTest::Listener").pointerTypes("ListenerInternal"));
```

The generated JNI code will end up scoping the references twice (example error):
```
javacpp_test/test/jniNested.cpp:297:101: error: no member named 'com' in 'com::company::NestedTest'; did you mean simply 'com'?
static void JavaCPP_test_Nested_00024NestedTest_00024ListenerInternal_deallocate(void *p) { delete (::com::company::NestedTest::com::company::NestedTest::Listener*)p; }
``` 
If the original class is in the global namespace, the rename works correctly.

I'm attaching a trivial example that recreates the issue.  Run build.sh in the root directory.

[javacpp_test.tar.gz](https://github.com/bytedeco/javacpp/files/3757110/javacpp_test.tar.gz)

**Comment by saudet** [comment]
Fixed in commit f901aa9025dc63f6e595c43479f98226c138bfb7. Thanks for reporting!

**Comment by saudet** [comment]
JavaCPP 1.5.2 has been released with the fix. Enjoy!
